src/main/java/com/faforever/client/player/FriendJoinedGameNotifier.java

```diff
@@ -49,11 +53,13 @@ void postConstruct() {
   @Subscribe
   public void onFriendJoinedGame(FriendJoinedGameEvent event) {
     Player player = event.getPlayer();
-    Game game = player.getGame();
+    Game game = event.getGame();
```

Comment:
Not saying it's wrong, but why did you decide to add the game to the event, instead of reading it from the user?

I do see one reason why this is better, but I'd like to know what yours is :)

Reply:
Well, I had some issues with how games were set to the player. After fixing that, I still think it's more reasonable and logical that the message about a player joining a game should contain exactly that information; the player and the game.